docs/admin/administration.md

```diff
@@ -534,6 +534,24 @@ The device will now be bound to a user-space driver (e.g. VFIO) and can be acces
 DAOS I/O engine processes (and used in the following `dmg storage replace nvme` command
 as a new device).
 
+- Once an engine is using a newly added (hot-inserted) SSD it can be added to the persistent
+NVMe config (stored on SCM) so that on engine restart the new device will be used.
+
+To update the engine's persistent NVMe config with the new SSD transport address, run the
+following command (replace SSD PCI address, engine index and hostname with appropriate values):
+```bash
+$ dmg storage nvme-add-device -a 0000:84:00.0 -e 0 -l wolf-167
+Command completed successfully
+```
+
+The optional [--tier-index|-t] command parameter can be used to specify which storage tier to
+insert the SSD into, if specified then the server will attempt to insert the device into the tier
+specified by the index, if not specified then the server will attempt to insert the device into
+the bdev tier with the lowest index value (the first bdev tier).
+
+The device will now be registered in the engine's persistent NVMe config so that when restarted,
+the newly added SSD will be used.
+
```

Comment:
what needs to be done with bdev definition in the daos_server.yml file? shouldn't it also be updated to the new NVMe ID?

Reply:
The daos_server.yml file is read-only from the perspective of the control plane, we intentionally don't write to the file as it's user editable and updating it auto-magically is likely to lead to confusing consequences. One option is to have a dmg dump config command which would output the running config and would have the updated bdev lists. The problem does need some more thinking about wrt keeping the engine nvme.conf in sync with control plane configurations and yaml files. @mjmac may have some other thoughts.